docs/sources/mimir/get-started/play-with-grafana-mimir/index.md

```diff
@@ -116,14 +116,13 @@ as a new set of time series. In this section you're going to configure a recordi
 offered by Grafana.
 
 1. Open [Grafana Alerting](http://localhost:9000/alerting/list).
-2. Click "New alert rule", which also allows you to configure recording rules.
+2. Click "New recording rule", which also allows you to configure recording rules.
```

Comment:
This sounds a bit redundant. Did you want to say that it also allows you to configure alert rules? If not, we can just leave it as "Click **New recording rule**."

Reply:
In Grafana Alert section, there are rules that users can make by user's choice

One is the alert rule, and the other is recording rule

So, in the tutorial, beginners would learn each rule once!


But I think both of the rules were written as all alert rules. So I was confused to learn first ly

I suggest modifying the word to "New recording rule" from "New Alerting rule"

Reply:
That sounds good. Then I think we can just say "Click **New recording rule**" and leave it at that.